example/lib/main.dart

```diff
@@ -4,6 +4,8 @@ import 'package:example/actions/gas_price.dart';
 import 'package:example/actions/write_contract.dart';
 import 'package:flutter/material.dart';
 import 'package:wagmi_flutter_web/wagmi_flutter_web.dart' as wagmi;
+import 'package:web3dart/crypto.dart';
```

Comment:
Would you agree to use the webthree lib (https://pub.dev/packages/webthree) instead of web3dart please?
Webthree is still maintained whereas web3dart has not been for months.
Webthree is a fork of web3dart with additional features for error handling, event pulling and EVM wallet connections.

Reply:
Sure resolved.

@redDwarf03 a word of advice, kindly be flexible enough in these small issues when we are trying to implement initial features. In the past, you asked us to not to use js_utils and blocked a merged for 2 days since it lacked wasm support, while this lib was itself using dart:html which lacks WASM support as well.

Kindly let all the necessary actions get implemented first - these small things can be resolved later .